### PR TITLE
UFORGE-12943

### PR DESCRIPTION
**Pull request by TimCanu**
As a UI user, I can be redirected to my appliance install profile when the appliance is "not ready"

### CODE REVIEW

File: ui-modules/blueprint-composer/app/components/util/d3-blueprint.js [hunk 1]
```diff
@@ -298,26 +298,32 @@ export function D3Blueprint(container, $scope) {
...
         let applianceName = node.data.miscData.get('typeName');
+        let warningButtonLink = node.data.miscData.get('config')[0].uriOSProfile;
+        let warningButtonMessage = "Choose OS Profile";
         let warningMessage;
         if(interactiveAppliances.includes(applianceName)) {
             warningMessage = "Requires additional user interactions at startup. Please edit the install profile.";
         } else if (noOsProfileAppliances.includes(applianceName)) {
             warningMessage = "This appliance has no OS profile. Please edit it.";
         }
-        $scope.$root.$broadcast("iconWarningMouseover", x, y, applianceName, warningMessage, node.data.miscData.nodeId);
```
Comment (TimCanu) [inline ui-modules/blueprint-composer/app/components/util/d3-blueprint.js:320]: Should not be removed

File: ui-modules/blueprint-composer/app/components/util/d3-blueprint.js [hunk 2]
```diff
@@ -298,26 +298,32 @@ export function D3Blueprint(container, $scope) {
         let x = d3.event.pageX + 5;
         let y = d3.event.pageY;
         let applianceName = node.data.miscData.get('typeName');
+        let warningButtonLink = node.data.miscData.get('config')[0].uriOSProfile;
+        let warningButtonMessage = "Choose OS Profile";
```
Comment (TimCanu) [inline ui-modules/blueprint-composer/app/components/util/d3-blueprint.js]: Don't initialize it?

File: ui-modules/blueprint-composer/app/components/not-ready-appliance/not-ready-appliance.directive.js [hunk 3]
```diff
@@ -36,10 +36,21 @@ export function notReadyApplianceDirective() {
...
+        controller: ['$scope','$location','blueprintSaverService',controller]
     };
 }
 
+function controller($scope, $location, blueprintSaverService) {
+    $scope.saveAndRedirect = saveAndRedirect;
+
+    function saveAndRedirect(path) {
+        blueprintSaverService.save(false);
+        $location.path( path );
```
Comment (TimCanu) [inline ui-modules/blueprint-composer/app/components/not-ready-appliance/not-ready-appliance.directive.js]: Code format?

**Comment by yanngit** [comment]
Done modifications

**Review by yanngit — APPROVED** [review]
Tim is ok

**Pull request merged**